src/qt/rpcconsole.cpp

```diff
@@ -834,12 +834,14 @@ void RPCConsole::message(int category, const QString &message, bool html)
     out += "<table><tr><td class=\"time\" width=\"65\">" + timeString + "</td>";
     out += "<td class=\"icon\" width=\"32\"><img src=\"" + categoryClass(category) + "\"></td>";
     out += "<td class=\"message " + categoryClass(category) + "\" valign=\"middle\">";
-    if(html)
+    if (html) {
         out += message;
-    else
+    } else {
         out += GUIUtil::HtmlEscape(message, false);
+    }
     out += "</td></tr></table>";
     ui->messagesWidget->append(out);
+    scrollToEnd();
```

Comment:
If the user has scrolled up, we shouldn't ignore that and reset scroll to the bottom...

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/123#issuecomment-727557347).